babel_util/recommenders/centrality.py

```diff
@@ -9,17 +9,15 @@ def __init__(self):
         self.edge_count = 0.0
 
     def parse_file(self, stream, delimiter='\t'):
-        for line in stream:
-            _, to_id = map(str.strip, line.split(delimiter))
+        for from_id, to_id in stream.split(delimiter):
```

Comment:
This likely isn't doing what you intend. Specifically, to_id will have a \n included. This means dictionary lookups won't operate correctly. The strip function will fix this problem.